spec/helpers/medications_helper_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+describe MedicationsHelper do
+  let(:current_user) { create(:user) }
+  let(:medication) { create(:medication, user: current_user) }
+
+  before(:each) do
+    @medication = medication
+  end
+
+  def get_field(field_name)
+    @fields.find {|f| f[:id] == field_name}
+  end
+
+  describe 'fields' do
+    before(:each) do
+      @fields = common_fields
+    end
+
+    it 'returns common medication fields' do
+      expect(get_field('medication_name')[:value]).to eq(@medication.name)
+      expect(get_field('medication_strength')[:value]).to eq(@medication.strength)
+      expect(get_field('medication_strength_unit')[:value]).to eq(@medication.strength_unit)
+      expect(get_field('medication_total')[:value]).to eq(@medication.total)
+      expect(get_field('medication_total_unit')[:value]).to eq(@medication.total_unit)
+      expect(get_field('medication_dosage')[:value]).to eq(@medication.dosage)
+      expect(get_field('medication_dosage_unit')[:value]).to eq(@medication.dosage_unit)
+      expect(get_field('medication_refill')[:value]).to eq(@medication.refill)
+      expect(get_field('medication_comments')[:value]).to eq(@medication.comments)
+    end
+
+    context 'when refill reminder is active' do
+      let(:medication) { create(:medication, :with_refill_reminder, user: current_user) }
+      it 'medication_refill_reminder is set' do
+        @medication = medication
```

Comment:
I think we can use `medication` as it is without converting it to an instance variable